conanfile.py

```diff
@@ -39,7 +39,7 @@ def configure(self):
                                             "but used %s" % (self.settings.arch, api_level))
         if self.settings.compiler.version != "8":
             raise ConanInvalidConfiguration("only Clang 8 is supported")
-        if self.settings.compiler.libcxx != "libc++":
+        if not str(self.settings.compiler.libcxx) in ["c++_shared", "c++_static"]:
```

Comment:
Is the `str` really needed here?

Reply:
yes, better keep it, just in case someone uses older conan release with older `settings.yml`